capstone/src/main/webapp/index.html

```diff
@@ -2,30 +2,30 @@
 <!doctype html>
 <html lang="en">
   <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <link href="style.css" type="text/css" rel="stylesheet">
-        <title>Home</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="style.css" type="text/css" rel="stylesheet">
+    <title>Home</title>
   </head>
   <body class="text-center">
-        <div class="welcome">
-            <h1 class="cover-heading">Welcome.</h1>
-            <p class="lead">
-                This website will give you election Information
-                relevant to you, and allow you add election dates to your 
-                Google Calendar.
-            </p>
-        </div>
+    <div class="welcome">
+        <h1 class="cover-heading">Welcome.</h1>
+        <p class="lead">
+            This website will give you election Information
+            relevant to you, and allow you add election dates to your
+            Google Calendar.
+        </p>
+    </div>
 
-        <br>
-
-        <div class="inputinfo">
-            <form class="form-input" action="/VoterServlet1">
-                <label for="inputEmail" class="sr-only">Enter Your Information Here:</label>
-                <input type="address" id="address" name="address" class="form-control" placeholder="Enter Address" required autofocus>
-                <input type="electionID" id="inputID" name="inputID" class="form-control" placeholder="Election ID" required>
-                <input type="submit" class="btn btn-lg btn-primary btn-block">
-            </form>
-        </div>
+    <br>
+      
+    <div class="inputinfo">
+        <form class="form-input" action="/VoterServlet1">
+            <label for="Title" class="sr-only">Enter Your Information Here:</label>
```

Comment:
You don't need "for='Title'" here
The attribute "for" binds the label to a form element & in this case, it's not binding it to anything